### ISSUE TITLE
Rollup 3

### ISSUE THREAD

**Issue by brettz9** [piotr-cz/rollup-plugin-prepend-modules#1]
Any chances this can be updated to work with Rollup 3 (or if it already works, to update `peerDependencies`)?

Thanks!

**Comment by piotr-cz** [comment]
Can help me out?

1. prepare pull request with
  ```json
  "peerDependencies": {
    "rollup": "^1.20.0 || ^2.0.0 || ^3.0.0"
  }
  ```
2. install the plugin from pull request branch in your project (given that you created branch `feature/rollup-3`)
  ```console
  npm install @piotr-cz/swr-idb-cache@github:brettz9/rollup-plugin-prepend-modules#feature/rollup-3",
  ```
3. update rollup to ^3 in your project
4. test if you can build the project

When checking the Rollup docs > [Changes to the Plugin API](https://rollupjs.org/guide/en/#changes-to-the-plugin-api) I can't see any breaking changes

Nowadays I'm using vite with [@preact/preset-vite](https://github.com/preactjs/preset-vite) which does what this plugin does for me.

**Comment by piotr-cz** [comment]
Fixed in [#62d0a76](https://github.com/piotr-cz/rollup-plugin-prepend-modules/commit/62d0a76d4eb733c51f2e99e3b50f764fbcd1b048)
